### PR TITLE
Nuclia Specs

### PR DESCRIPTION
**Pull request by clayrisse**
Hello folks
We think this is a cool project putting together all this information. 
All the luck and here is our 2 cents ;)

### CODE REVIEW

File: docs/tools/vdb_table/data/nucliadb.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "slug": "nucliadb"
   },
   "oss": {
-    "support": "",
+    "support": "full",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/nucliadb.json:12]: please provide the github repo link in the source_url here

File: docs/tools/vdb_table/data/nucliadb.json [hunk 2]
```diff
@@ -110,43 +110,43 @@
...
     "source_url": "",
     "comment": ""
   },
   "doc_size": {
     "bytes": 0,
-    "unlimited": false,
+    "unlimited": true,
     "source_url": "",
-    "comment": ""
+    "comment": "Max vectors of 50K"
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/nucliadb.json:145]: is this referring to number of vectors in an index? or something else?

**Review by dhruv-anand-aintech — CHANGES_REQUESTED** [review]
Please provide references to code/docs/example notebook in the source_url for each support claim.

The definition of each attribute is in https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/vendor.schema.json

File: docs/tools/vdb_table/data/nucliadb.json (hunk 1 above)
Comment (clayrisse) [reply docs/tools/vdb_table/data/nucliadb.json:12]: Repo link added

File: docs/tools/vdb_table/data/nucliadb.json (hunk 2 above)
Comment (clayrisse) [reply docs/tools/vdb_table/data/nucliadb.json]: More detail explanation has been provided

**Comment by svonava-superlinked** [comment]
@clayrisse I'd love to get the changes in - did you close the PR intentionally?

**Comment by svonava-superlinked** [comment]
> @clayrisse I'd love to get the changes in - did you close the PR intentionally?

Oh nevermind, just saw the other updated PR. Thanks!

**Pull request closed without merging**